plugins/crop-circle-tracker

```diff
@@ -0,0 +1,2 @@
+repository=https://github.com/mattjrumble/crop-circle-tracker-plugin.git
+commit=d1ef0dcd56629cd3bf5ecb5980dd0c85a1e9dd22
```

Comment:
Since your plugin defaults to sending data to an external server, it requires a `warning` field.
```suggestion
commit=d1ef0dcd56629cd3bf5ecb5980dd0c85a1e9dd22
warning=This plugin submits your IP address to a server not controlled or verified by the RuneLite developers.
```